engine/Shopware/Models/Document/Document.php

```diff
@@ -56,6 +56,14 @@ class Document extends ModelEntity
     private $name = '';
 
     /**
+     * Contains a unique identifier for plugins
+     *
+     * @var string
+     * @ORM\Column(name="identifier", type="string", nullable=false, unique=true)
+     */
+    private $identifier = '';
```

Comment:
I'm not happy with this naming, because the model already has a unique identifier called `id`. In my opinion the chosen wording will lead to misunderstandings and confusion. Perhaps we should use something like `alias`, which in my opinion expresses the intended meaning of the new property more precisely.